src/pybind/mgr/cephadm/tests/fixtures.py

```diff
@@ -37,16 +37,41 @@ def mon_command(*args, **kwargs):
 @pytest.yield_fixture()
 def cephadm_module():
     with mock.patch("cephadm.module.CephadmOrchestrator.get_ceph_option", get_ceph_option),\
-            mock.patch("cephadm.module.CephadmOrchestrator.remote"),\
-            mock.patch("cephadm.module.CephadmOrchestrator.send_command"), \
-            mock.patch("cephadm.module.CephadmOrchestrator.mon_command", mon_command):
+         mock.patch("cephadm.module.CephadmOrchestrator.remote"), \
+         mock.patch("cephadm.module.CephadmOrchestrator.send_command"), \
+         mock.patch("cephadm.module.CephadmOrchestrator.get_osdmap"), \
+         mock.patch("cephadm.module.CephadmOrchestrator.mon_command", mon_command):
```

Comment:
the indentation was intentional to have it visually distinct from the block below

Reply:
ok, reverting